server/repository/repositories.go

```diff
@@ -0,0 +1,77 @@
+package repository
+
+//TODO: use "user.go" as a example, not this one ;)
+
+import (
+	"fmt"
+
+	"github.com/src-d/code-annotation/server/model"
+)
+
+var assignments []*model.Assignment
+
+// ErrNoAssignmentsInitialized is the error returned when are requested the Assignments of a User
+// for a given Experiment, but they has not been yet created
+var ErrNoAssignmentsInitialized = fmt.Errorf("No assignments initialized")
```

Comment:
is the error returned when **the Assignments of a User** are requested
for a given Experiment, but they **have** not been yet created